src/test/java/com/sfuentes/EmployeeTest.java

```diff
@@ -0,0 +1,84 @@
+package com.sfuentes;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class EmployeeTest {
+
+  @Test
+  void test_exceptionIncorrectAmountOfSales() {
+    double[] sales = {100.00, 100.00, 100.00, 100.00};
+
+    Exception exception = assertThrows(IllegalArgumentException.class, () -> {
+      Employee test = new Employee(1, sales);
+    });
+
+    String expectedMessage = "Incorrect amount of sales";
+    String actualMessage = exception.getMessage();
+
+    assertTrue(actualMessage.contains(expectedMessage));
+  }
+
+  @Test
+  void calculateSumSalary_integer() {
+    double[] sales = {100.00, 100.00, 100.00};
+    Employee test = new Employee(1, sales);
+    double expected = 30.00;
+    double actual = test.calculateSumSalary();
+
+    assertEquals(expected, actual);
+  }
+
+  @Test
+  void calculateSumSalary_decimal() {
```

Comment:
for this test and the following ones please be more descriptive about the state you are testing:
This is a common pattern used to name tests, you can omit the expected behavior if you feel so:

MethodName_StateUnderTest_ExpectedBehavior: There are arguments against this strategy that if method names change as part of code refactoring than test name like this should also change or it becomes difficult to comprehend at a later stage. Following are some of the example:
isAdult_AgeLessThan18_False
withdrawMoney_InvalidAccount_ExceptionThrown
admitStudent_MissingMandatoryFields_FailToAdmit

Reply:
fixed